libs/common/bit_array.go

```diff
@@ -412,6 +412,7 @@ func (bA *BitArray) UnmarshalJSON(bz []byte) error {
 			bA2.SetIndex(i, true)
 		}
 	}
-	*bA = *bA2
+	// assignment copies lock value to *bA: github.com/tendermint/tendermint/libs/common.BitArray contains sync.Mutex
+	*bA = *bA2 //nolint:govet
```

Comment:
ok to copy the lock?

Reply:
I think  it is OK here: the object is just being constructed ... If you want to remove the `nolint`, you could get rid of `bA2` here instead and init the fields of `bA` directly:
```go
        bA.Bits = numBits
	bA.Elems = make([]uint64, (numBits+63)/64)
	for i := 0; i < numBits; i++ {
		if bits[i] == 'x' {
			bA.SetIndex(i, true)
		}
	}
```

Reply:
no, I think it's fine to leave nolint